app/views/api/policies/edit.html.slim

```diff
@@ -3,4 +3,4 @@
 = semantic_form_for @proxy, url: admin_service_policies_path(@service) do |f|
   = render 'api/integrations/apicast/shared/policies'
   = f.buttons do
-    = f.commit_button t('formtastic.actions.policies.update'), button_html: {class: 'important-button', name: 'update', id: 'policies-button-sav', title: 'Update Policies'}
+    = f.commit_button t('formtastic.actions.policies.update'), button_html: {class: 'disabled-button', name: 'update', id: 'policies-button-sav', title: 'Update Policies', disabled: true}
```

Comment:
How about making the value of this button "Update Policy Chain" so it's double clear "Update Policy" is one thing, "Update Policy Chain" is another?